geckodriver.log

```diff
@@ -0,0 +1 @@
+1624633224198	geckodriver	INFO	Listening on 127.0.0.1:54777
```

Comment:
We don't want to track this log with the project, we should add this to your .gitignore (and remove the current tracking, with `git rm --cached geckodriver.log`)